### DISCUSSION TITLE
Port code to run on router

### DISCUSSION THREAD

**Discussion by blixten85** [mitmproxy/mitmproxy#6192] [General]
I want to run this program in my router ASUS RT-AX86U. This router can run Asus-Merlin firmware and thus execute binary files. Install applications from opkg repository and such. 

https://www.asuswrt-merlin.net/[https://www.asuswrt-merlin.net/](https://www.asuswrt-merlin.net/)

**Comment by ThomasBucaioni** [comment]
The binary `mitmproxy` should work out of the box, you can download it here: https://mitmproxy.org/
